source/slang/hlsl.meta.slang

```diff
@@ -442,26 +460,44 @@ void asuint(double value, out uint lowbits, out uint highbits);
 void asuint(double value, out uint lowbits, out uint highbits);
 
 // Reinterpret bits as a uint (HLSL SM 4.0)
+
+// GLSL Scalar
```

Comment:
This comment makes me wonder (it is hard to tell from the diffs) if this change is also duplicating out the declarations so that we have distinct declarations of these operations for HLSL vs. GLSL. I can understand why that is nice for clarity, but the resulting bloat in the size of the stdlib if we did that for every single function makes me nervous.

Reply:
It does leave the HLSL definitions that were previously there. That because of the behavior, perhaps they can be removed, but the behavior seems intuitive and is not quite rich enough to describe what you'd ultimately want, and somewhat verbose when supporting other targets. So as previously discussed we should look at improving/documenting.